public.tf

```
@@ -62,6 +65,18 @@ resource "aws_route" "public" {
   }
 }
 
+resource "aws_route" "public-ipv6" {
```

dashes in resource names should be underscores
```suggestion
resource "aws_route" "public_ipv6" {
```